extensions/prototypes/labgraph_monitor/src/redux/reducers/ws/interfaces/IGraph.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import INode from './INode';
+
+interface IGraph {
```

Comment:
It seems like this interface is present in multiple files. Any way we could just have one central source for the `IGraph, INode` interfaces, rahter than multiple `IGraph.ts` files?

Reply:
Done